### DISCUSSION TITLE
"Skipping remote @import of [Google Font] as resource is not allowed."

### DISCUSSION THREAD

**Discussion by Inayatullahsh** [tailwindlabs/tailwindcss#3640] [Help]
Can someone help me why I am getting this `WARNING: Skipping remote @import of "https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700&family=Pacifico&display=swap" as resource is not allowed.` error when I run `npm run build`?
Note: I am using Tailwind CSS with Django.
@adamwathan 

My _tailwind.config.js_ file.
![carbon (1)](https://user-images.githubusercontent.com/47157614/108893811-23bc1000-762f-11eb-9c2c-8b4c4f02c61c.png)

**Comment by simonswiss** [comment]
Hey!

Can you make sure the `@import` statement is at the top of your CSS file? Also, you might need to use the [`postcss-import`](https://tailwindcss.com/docs/using-with-preprocessors#build-time-imports) plugin.

**Comment by panfanky** [comment]
Using `postcss-import` and @import statements are on top of the style.pcss file and still I get this message
`gulp-postcss: style.pcss
clean: Skipping remote @import of "https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Roboto+Condensed:wght@300;400;700&family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400&display=swap" as resource is not allowed.
`
